README.md

```diff
@@ -111,6 +111,26 @@ while the Trace SDK applies the plugin.
 Trace is released under the MIT license. See 
 [LICENSE](https://github.com/bitrise-io//trace-android-sdk/blob/main/LICENSE.md) for details.
 
+## Code quality
+
+### Checkstyle
+
+trace-android-sdk uses Checkstyle (<https://checkstyle.sourceforge.io>) to adhere our coding 
+standards. All included builds and subprojects use it, except 'opencensus', because that is a 
+generated module.
+
+Application builds will fail on the CI if there is an issue. The coding standards are 
```

Comment:
nit: What do you mean by application builds will fail on the CI? How is that different from a standard build? possibly the word "application" just needs to be removed and "builds will fail on the ci if there is any issues with the checkstyle?

Reply:
Every build on Bitrise is usually referred as "Bitrise App Build". I see, that this can be confusing, I can remove the application word if it helps?